hack/shellcheck-test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-  
+
```

Comment:
Separate commit, can be the same as the shellcheck error fix

Reply:
done